app/controllers/posts_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
   end
 
   def index
-    @posts = Post.all.order(updated_at: :desc)
+    @posts = Post.all.order(created_at: :desc)
```

Comment:
Why did you choose to change the order posts were displayed at from `updated_at` to `created_at`?